### ISSUE TITLE
XML output: table caption doesn't have ID

### ISSUE THREAD

**Issue by canton7** [doxygen/doxygen#8932]
**Describe the bug**

Doxygen lets you specify a table caption ID with:

```html
<table>
<caption id="multi_row">Complex table</caption>
...
</table>
```

and then refer to the ID using `\ref`.

However, the caption ID isn't emitted anywhere in the XML. In compound.xsd, `docTableType` is given as:

```xml
<xsd:complexType name="docTableType">
  <xsd:sequence>
    <xsd:element name="caption" type="docCaptionType" minOccurs="0" maxOccurs="1" />
    <xsd:element name="row" type="docRowType" minOccurs="0" maxOccurs="unbounded" />
  </xsd:sequence>
  <xsd:attribute name="rows" type="xsd:integer" />
  <xsd:attribute name="cols" type="xsd:integer" />
  <xsd:attribute name="width" type="xsd:string" />
</xsd:complexType>
```

and the `docCaptionType` is given as:

```xml
<xsd:complexType name="docCaptionType" mixed="true">
  <xsd:group ref="docTitleCmdGroup" minOccurs="0" maxOccurs="unbounded" />
</xsd:complexType>
```

There's no space for the caption ID. I get a `<ref refid="group___test_1multi_row" kindref="member">Complex table</ref>`, but the ID `group___test_1multi_row` doesn't appear anywhere else, so I end up with a link to something non-existent.

**Expected behavior**

The XML output should contain a table caption's ID.

**To Reproduce**

```c
/**
 * <table>
 * <caption id="multi_row">Complex table</caption>
 * <tr>
 *  <td>Cell 1</td><td>Cell 2</td>
 * </tr>
 * </table>
 * 
 * A ref to \ref multi_row
 */
```

**Version**

1.9.2 (caa4e3de211fbbef2c3adf58a6bd4c86d0eb7cb8) on Windows 64-bit, from the installer on the website.  

Thanks!

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #8938

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.3.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
